wiki/en/Running-a-Server.md

```diff
@@ -18,50 +18,39 @@ permalink: "/wiki/Running-a-Server"
+Usually, problems are on the _Client_ side and should be fixed there.
+Various problems can arise when setting up Servers - especially when run on a low-bandwidth home connection.
+Therefore:
+* Use other peoples' (public) Servers at first to isolate any issues that may in fact be due to Client configuration.
+* Ensure you have enough bandwidth: Less than 5 players on a slower-speed home connection (eg 10 Mbit/s down and 1 Mbit/s up) are fine. You can read more about network requirements at [different quality settings here](Server-Bandwidth).
 
 ### In general
 
-- Consider using a cloud host to get better ping times if you're having problems
-
-- Any Server should have at least 1.6GHz CPU frequency and 1GB RAM
-
-- Running a Server may require you to adjust any firewalls running on or outside of your machine or cloud host.
+* Consider using a cloud host, not your home internet connection to get better ping times if you're having problems
+* Any Server should have at least 1.6GHz CPU frequency and 1GB RAM
+* Running a Server may require you to adjust any firewalls running on or outside of your machine or cloud host.
+* Running an Uregistered Server at home will require you to [port forward](#running-an-unregistered-server) on your router. When running a Registered Server, port forwarding should not be necessary in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode.
+* Jamulus only has limited IPv6 support which needs to be enabled with a command line argument on the Client and Server. There are plans to expand IPv6 support.
 
-- Running an Uregistered Server at home will require you to [port forward](#running-an-unregistered-server) on your router. When running a Registered Server, port forwarding should not be necessary in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode.
-
-- Jamulus only has limited IPv6 support which needs to be enabled with a command line argument on the Client and Server. There are plans to expand IPv6 support.
-
-# Server Types
+## Server Types
 
 You can run your Server in different ways (either at home or on a 3rd party host):
 
 ### 1. Registered
 
-Your Server will be listed in a Directory. By default, Jamulus has a list of Public Directories that Clients can connect to. If you register with one of these, anyone can then discover and connect to your Server. You can also have your Server listed on a Custom Directory, if that better meets your needs. 
+Your Server will be listed in a Directory. By default, Jamulus has a list of Public Directories that Clients can connect to. If you register with one of these, anyone can then discover and connect to your Server. You can also have your Server listed on a Custom Directory, if that better meets your needs.
```

Comment:
Mm, not pressing for this but "built in" rather than "Public" is my current preference...  "Built in" works against "Custom" better than "Public" against "Custom".

Reply:
Ok. Changed it. But I think it's not much different.